src/iuse.cpp

```diff
@@ -4954,24 +4954,6 @@ int iuse::arrow_flamable(player *p, item *it, bool, const tripoint& )
     return 1;
 }
 
-int iuse::molotov(player *p, item *it, bool, const tripoint& )
-{
-    if (p->is_underwater()) {
```

Comment:
Ah, I see, code in ```iuse_actor.cpp:L69``` prevents any transform action underwater. Good to know that.